packages/block-library/src/site-title/index.js

```diff
@@ -15,6 +15,9 @@ export { metadata, name };
 
 export const settings = {
 	title: __( 'Site Title' ),
+	description: __(
+		'The name of the site is an important setting that determines how the website is displayed in various places, including the title bar of a web browser, search engine results, and RSS feeds. It can also be edited in Settings > General.'
```

Comment:
The name of the site is an important setting that determines how the website is displayed in various places, including the title bar of a web browser, search engine results, and RSS feeds. It can also be edited in Settings > General.'

Because many of the core blocks start with words like "Introduce," "add," and "Insert," perhaps something similar could work here:

> Display the title of your site — perfect for your site header.